### ISSUE TITLE
CORS blocked,failed 

### ISSUE THREAD

**Issue by Dhinendran** [aws/chalice#970]
1.Failed to load resource: the server responded with a status of 500 ()
2.Access to XMLHttpRequest at 'https://apixxxx.amazonaws.com/api/getuserdetails' from origin 'http://localhost:4200' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

working in local but while working with Api its shows error

**Comment by lstroud** [comment]
+1

**Comment by kyleknap** [comment]
@Dhinendran or @lstroud could you provide a sample app and steps to perform to help reproduce this? It is a little hard to follow what may be causing this.
